content/docs/guide/gamemodes/climb.md

```diff
@@ -8,19 +8,42 @@ tags:
+
+Sometimes you may not be able to take a direct line due to an obstruction. The 2 most common forms of this is rounding a corner (often without being able to see the next platform) and 2 platforms separated by a thin wall. 
+
+In this case, air strafing is not used to gain speed but as a form of air control. The principle is the same but typically smooth turns at the cost of speed are needed (don't take out all your speed or you'll fall straight down!).
+
+### Bhop
+depending on the physics system used (1.6, CSGO Vanilla, KZT) and map design, bhopping can have reduced potential compared to the bhop game mode. In all physics systems it can be used as a way to preserve speed gains or minimize speed loss. 
+Bhops can be used as a general speedrunning technique by stringing jumps together that are close in distance and height. 
+The map can also encourage a bhop by making a very small platform that, if stopped on, does not allow the player to get a running start before they jump to the next platform. 
+
+1.6 Climb has a jump-landing penalty, the impact of which can be severely limited with bhops. 
+
+### Surf
+Surf/Slide is also limited compared to its dedicated game mode. Air acceleration in Climb  (10aa?) is a lot less than surf's(150aa?)
+This means players can't as easily flick onto a ramp ("board") and genearate speed, or flick off a ramp and generate height. surfs are usually brief and focus on keeping on the board (**"sliding"**) rather than gaining enough speed to fly off. However, the process of surfing is still the same as its game mode counterpart.
+In 1.6 Climb, Surf has the additional quirk of allowing the user to slide up gentle slopes with just 'W' while looking at the slope
+
+
+### Stamina (Jump-Land Penalty) [1.6 Climb Only]
+Any time the player jumps (not when they are in the air for any other reason such as falling off something) a timer starts for about 1.31 seconds. 
+Whenever the player lands, they will be slowed for however much time is left on the timer.
+For reference, if a player jumps and lands on flat ground about 0.66 seconds or about half the time is taken.
+This means that jumping up to something slows you down longer since you land earlier. And jumping to something beneath will slow you less or not at all.
+For speedrunning getting around this stamnia is a top prority for optimzing runs. For completion players only certain types of platforms make stamina a cause for concern namely ones that are to small or awkward to get a running start from a standsill.
+[This doc goes into futher detail](https://kzguide.gitlab.io/techniques/stamina/).
 
```

Comment:
very good explanation of stamina!
as a closing statement, or paragraph maybe could say that there's lots of other techniques and mechanics in 1.6 kz, like highjumps (edge friction), doubleduck and its derivatives (gstrafing, countjump), also ladders (everyone loves ladders!) which very rarely show up in other gamemodes. up to you ofc :)